doc/craft/ratelimitpolicy.md

```diff
@@ -0,0 +1,255 @@
+# Crafting A RateLimitPolicy
+A detail explanation of the fields in the RateLimitPolicy is outlined here.
+For a quick reference refer to [Reference](/kuadrant-operator/doc/reference/ratelimitpolicy/) section.
+
+## apiVersion and kind
+The latest version of the RateLimitPolicy is `kuadrant.io/v1beta2`.
+With the kind being `RateLimitPolicy`.
+```yaml
+apiVersion: kuadrant.io/v1beta2
+kind: RatelimitPplicy
+```
+
+## Metadata
+When creating the RateLimitPolicy the policy must be crated in the same namespace as the resource being targeted.
+```yaml
+metadata:
+    name: toystore
```

Comment:
Any special reason to be using 4-space indentation? I think most (if not all) our examples use 2 spaces.

Reply:
No special reason, my editor has a default of 4 spaces. I always found it as a good guide to highlight deeply nested code block and the cyclomatic complexity that brings.